docs/source/private_data_tutorial.rst

```diff
@@ -256,16 +249,26 @@ private data.
 :guilabel:`Try it yourself`
 
 Before installing, defining, and using the marbles private data chaincode below,
-we need to start the BYFN network. For the sake of this tutorial, we want to
-operate from a known initial state. The following command will kill any active
+we need to start the Fabric test network. For the sake of this tutorial, we want
+to operate from a known initial state. The following command will kill any active
 or stale docker containers and remove previously generated artifacts.
```

Comment:
```suggestion
or stale Docker containers and remove previously generated artifacts.
```

Reply:
Done, found somewhere else it was lowercase...